src/components/EnlargedPreview/index.js

```diff
@@ -30,12 +33,18 @@ class EnlargedPreview extends Component<Props, State> {
   }
 
   componentWillUpdate(nextProps: Props, nextState: State) {
+    if (nextState.isExpanded !== nextProps.isNavigationDisabled) {
```

Comment:
Minor: is the if safeguard really needed? any issue with setting something with the same value?

Reply:
No issue apart from polluting Redux with an unneeded event. It might be either with _if_ statement or hardly switching it off to avoid any race conditions (which I didn't see that could happen).